apps/indexer/src/api/controller/token-distribution/controller.ts

```diff
@@ -15,6 +15,10 @@ export function tokenDistribution(
   app: Hono,
   repository: TokenDistributionRepository,
 ) {
```

Comment:
The loop with console.log seems like a placeholder or debugging code. Be sure to remove or replace this with meaningful logic before proceeding to production. Additionally, consider adding comments or extracting this logic into a reusable function if it's part of future work. Overall, ensure that debugging statements are cleaned up to maintain clean, maintainable code.